nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/grok/GrokRecordReader.java

```diff
@@ -89,6 +93,7 @@ public Record nextRecord() throws IOException, MalformedRecordException {
         // the next call to nextRecord()) or we will attach it to the previously read record.
         String stackTrace = null;
         final StringBuilder toAppend = new StringBuilder();
+        final StringBuilder raw = new StringBuilder(line);
```

Comment:
I think this needs to be updated slightly based on some changes in master, looks like the beginning of this method changed a little bit and the 'line' variable is no longer visible here